### DISCUSSION TITLE
Does waking up the thread network take as long as restarting it

### DISCUSSION THREAD

**Discussion by suc1** [openthread/openthread#10792] [Q&A]
Hi community,

We are going to develop a large-scale Thread network with the goal of **minimizing power consumption** as much as possible.

One design is to have the Border Router & Leader always stay awake, while other Routers and End Devices enter sleep mode. 
When a message needs to be sent, the upper-layer software calculates the path, wakes up the Routers and End Devices on  the path, then sends the message.

The question is, **does waking up the network take as long as restarting it**?

For example:
[Espressif Large-Scale Thread Network Performance Demonstration Based on ESP Thread SoCs](https://www.youtube.com/watch?v=0WXcu_r_lvQ)
It gives a figure: a 40-node network takes 30 seconds to form.
In a network with 300 nodes, if only 40 nodes will be woken up, does it still take 30 seconds to form the network?

Thank you very much

**Comment by jwhui** [comment]
Thread does not currently save routing information across reboots. As a result, it will require establishing a new topology after reboot.
